src/main/java/dk/kb/cfutv/service/CfuTvService.java

```diff
@@ -266,52 +261,47 @@ private String getYouSeeChannelId(String sBChannelId, Date date) throws ServiceE
         }
     }
 
-    private Date convertToUTC(Date date){
-        Locale locale = Locale.getDefault();
-        DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT,locale);
+    private ZonedDateTime convertToUTC(ZonedDateTime date){
+        return date.withZoneSameInstant(ZoneId.of("UTC"));
 
-        formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
-        String utctime = formatter.format(date);
-        Date utcDate = new Date(utctime);
-        return utcDate;
     }
 
     /**
      * Translates a date to a String that looks like part of the url needed to access the download web page.
      * @param date to be translated.
      * @return String that looks like part of the url needed to access the download web page.
      */
-    private String dateToUrlPart(Date date){
+    private String dateToUrlPart(ZonedDateTime date){
```

Comment:
The method as a whole could probably benefit from a re-write where the date formatting is handled by a date formatting class. Likely out of scope of here, but would be an improvement over the current implementation